server/app.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const express = require('express');
+
+let app = express();
+
+app.use(express.static(path.join(__dirname, '../public')));
+
+app.get('/', function (req, res) {
+  res.send();
+});
+
+app.get('/restaurant/:restaurant_id', function(req, res) {
```

Comment:
Are you not using /restaurant/:restaurant_id/reservations as your endpoint? Just clarifying if you meant to leave it off or not.